package.json

```diff
@@ -1,20 +1,23 @@
 {
   "name": "react-render-server",
-  "version": "0.4.0",
+  "version": "0.5.0",
   "description": "A node.js server for server-side rendering Khan Academy react components.",
-  "main": "src/main.js",
+  "main": "dist/main.js",
   "scripts": {
-    "start": "ulimit -n 999999 || ulimit -n 2048; node src/main.js -p 8080 --log-level=debug",
-    "dev": "nodemon -i src/render_cache src/main.js -- --dev",
-    "debug": "nodemon -i src/render_cache src/main.js -- --dev --log-level=debug",
-    "profile": "node-debug -p 8040 --no-preload src/main.js --dev",
-    "serve_local": "nodemon -i src/render_cache src/main.js -- --dev",
-    "test": "NODE_ENV=test mocha --exit --reporter spec 'src/*_test.js'",
+    "clean": "rm -rf dist",
+    "start": "ulimit -n 999999 || ulimit -n 2048; npm run build && node dist/main.js -p 8080 --log-level=debug",
+    "dev": "npm run build && nodemon dist/main.js -- --dev",
+    "debug": "npm run build && nodemon dist/main.js -- --dev --log-level=debug",
+    "profile": "npm run build && node-debug -p 8040 --no-preload src/main.js --dev",
+    "serve_local": "npm run build && nodemon dist/main.js -- --dev",
+    "test": "NODE_ENV=test mocha --exit --reporter spec 'src/*_test.js' --compilers js:@babel/register",
     "deploy": "./deploy.sh",
     "deploy:docker": "DOCKER=1 ./deploy.sh",
     "set_default": "./set_default.sh",
-    "lint": "npx eslint --config ./eslint/eslintrc src",
-    "prettyquick": "npx pretty-quick"
+    "lint": "eslint --config ./eslint/eslintrc src",
+    "prettyquick": "pretty-quick",
+    "build": "npm run clean && babel src --out-dir dist --source-maps --ignore src/testdata/**/*.js,*_test.js",
```

Comment:
Ah - it looks like the ignore eventually worked!?

Reply:
No, I just left it, like you suggested. I should delete the attempt to ignore, since it isn't working.